common/ZKLoginController.m

```diff
@@ -22,7 +22,7 @@
 #import "ZKLoginController.h"
 #import "credential.h"
 #import "zkSforceClient.h"
-#import "zkSoapException.h"
```

Comment:
Typo ?

Reply:
@superfell  Yep, good catch. Just fixed in the PR